src/platforms/python/guides/django/index.mdx

```diff
@@ -5,6 +5,8 @@ redirect_from:
   - /platforms/python/django/
 ---
 
+<PlatformContent includePath="getting-started-primer" />
+
 The Django integration adds support for the [Django Web Framework](https://www.djangoproject.com/)
```

Comment:
This information is partially duplicated in the primer, particularly the minimum required Django version